cli/cmd/agent.go

```diff
@@ -0,0 +1,444 @@
+//
+// Author:: Salim Afiune Maya (<afiune@lacework.net>)
+// Copyright:: Copyright 2020, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/lacework/go-sdk/api"
+	"github.com/lacework/go-sdk/lwrunner"
+	"github.com/olekukonko/tablewriter"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	tokenUpdateEnable  bool
+	tokenUpdateDisable bool
+	tokenUpdateName    string
+	tokenUpdateDesc    string
+
+	agentCmd = &cobra.Command{
+		Use:   "agent",
+		Short: "manage Lacework agents",
+		Long: `Manage agents and agent access tokens in your account.
+
+To analyze application, host and user behavior, Lacework uses a lightweight agent,
+which securely forwards collected metadata to the Lacework cloud for analysis. The
+agent requires minimal system resources and runs on most 64-bit Linux distributions.
```

Comment:
```suggestion
To analyze application, host, and user behavior, Lacework uses a lightweight agent,
which securely forwards collected metadata to the Lacework cloud for analysis. The
agent requires minimal system resources and runs on most 64-bit Linux distributions.
```

Reply:
@afiune Did you pull this text from support docs, or was it created new for the CLI?

Reply:
From https://support.lacework.com/hc/en-us/articles/360005230014-Supported-Operating-Systems